### ISSUE TITLE
Yahoo.finance.quotes not exist

### ISSUE THREAD

**Issue by ssitdikov** [scheb/yahoo-finance-api#12]
Hello.

Table yahoo.finance.quotes doesn't exist (renamed or changed to pm.finance). But pm.finance needed in additional permissions.

Maybe you know how to auth webapp in yql app by api?

**Comment by shlomnissan** [comment]
Same question here - `yahoo.finance.quotes` is no longer available :/

**Comment by scheb** [comment]
Yahoo decided to shut down the API without any warning. I doubt it will come back. See:
https://forums.yahoo.net/t5/Yahoo-Finance-help/http-download-finance-yahoo-com-d-quotes-csv-s-GOOG-amp-f/td-p/387096
and: https://download.finance.yahoo.com/d/quotes.csv?s=GOOG&f=d1ohgl1a2

**Comment by lesaff** [comment]
Here's the new API endpoint for Yahoo Finance
`https://query1.finance.yahoo.com/v7/finance/quote?symbols=GOOG`

Output
```
{"quoteResponse":{"result":[{"language":"en-US","quoteType":"EQUITY","quoteSourceName":"Delayed Quote","currency":"USD","fiftyTwoWeekLowChange":298.21002,"fiftyTwoWeekHighChange":-22.640015,"fiftyTwoWeekHighChangePercent":-0.021595031,"fiftyTwoWeekLow":727.54,"fiftyTwoWeekHigh":1048.39,"trailingPE":37.174282,"epsTrailingTwelveMonths":27.593,"epsForward":41.28,"fiftyTwoWeekLowChangePercent":0.40988815,"market":"us_market","exchangeDataDelayedBy":0,"tradeable":true,"marketState":"POST","regularMarketPrice":1025.75,"regularMarketTime":1510606801,"regularMarketChange":-2.3199463,"regularMarketOpen":1023.42,"regularMarketDayHigh":1031.58,"regularMarketDayLow":1022.57,"regularMarketVolume":763204,"sharesOutstanding":349479008,"bookValue":214.015,"fiftyDayAverage":987.32544,"fiftyDayAverageChange":38.42456,"fiftyDayAverageChangePercent":0.03891783,"twoHundredDayAverage":949.45764,"twoHundredDayAverageChange":76.29236,"twoHundredDayAverageChangePercent":0.08035362,"marketCap":717882392576,"exchange":"NMS","forwardPE":24.848597,"priceToBook":4.792888,"sourceInterval":15,"exchangeTimezoneName":"America/New_York","exchangeTimezoneShortName":"EST","gmtOffSetMilliseconds":-18000000,"postMarketChangePercent":-0.02437241,"postMarketTime":1510610206,"postMarketPrice":1025.5,"postMarketChange":-0.25,"regularMarketChangePercent":-0.22566034,"regularMarketPreviousClose":1028.07,"bid":1023.31,"ask":1025.52,"bidSize":1,"askSize":1,"messageBoardId":"finmb_29096","fullExchangeName":"NasdaqGS","longName":"Alphabet Inc.","financialCurrency":"USD","averageDailyVolume3Month":1326220,"averageDailyVolume10Day":1061300,"shortName":"Alphabet Inc.","priceHint":2,"symbol":"GOOG"}],"error":null}}```

**Comment by scheb** [comment]
@lesaff Thanks for sharing! I've updated the API client to use that new endpoints.

I've released a new major version 3.0 because of some breaking changes. Quote entity changed according to the new quotes data and exchange rates are now also represented as Quote.
